django/template/base.py

```diff
@@ -415,14 +419,33 @@ def __repr__(self):
             self.verbatim,
         )
 
+    # RemovedInDjango70Warning: When the deprecation ends, remove this
+    # and remove TEMPLATE_TAGS_MULTILINE from settings.py-tpl
+    @property
+    def tag_re(self):
+        from django.conf import settings
+
+        multiline = getattr(settings, "TEMPLATE_TAGS_MULTILINE", False)
```

Comment:
We should add this to the global settings fallback in `global_settings.py`. (That also has a side benefit of not needing to `getattr` here, but consistency is the bigger reason.)